esrally/time.py

```diff
@@ -51,14 +51,17 @@ def sleep(seconds):
     time.sleep(seconds)
 
 
-def _to_datetime(val, date_format=None):
+def _to_datetime(val, date_format=None, default_tz=timezone.utc):
```

Comment:
As this is a module-internal function I think it would make sense to expose this parameter also in the function `days_ago` which is using it.